ethclient/ethclient_test.go

```diff
@@ -83,7 +83,7 @@ var genesis = &core.Genesis{
 }
 
 var genesisForHistorical = &core.Genesis{
-	Config:    params.OptimismTestConfig,
+	Config:    params.OptimismTestCliqueConfig,
```

Comment:
This seems a bit out of scope for this PR? And see other comment about whether it is intended.

Reply:
Intended, this particular test needs a bedrock block > 0. In fact, it is this test for which the `OptimismTestCliqueConfig` was created (I just renamed it).